rtypes/item-collection.raml

```diff
@@ -74,6 +74,11 @@
             text/plain:
               example: |
                 "unable to update <<resourcePathName|!singularize>> -- malformed JSON at 13:4"
+        409:
+          description: "Optimistic locking failed"
```

Comment:
The locking successfully rejected the update, it didn't failed.
```suggestion
          description: "Optimistic locking version conflict"
```

Reply:
4XX are failures. But whatever.